packages/bot-web-ui/src/stores/save-modal-store.ts

```diff
@@ -160,14 +160,14 @@ export default class SaveModalStore implements ISaveModalStore {
             this.setButtonStatus(button_status.COMPLETED);
         }
 
+        this.updateBotName(bot_name);
```

Comment:
nitpick: add line break after 
```suggestion
        this.updateBotName(bot_name);
        
```